src/afrey/boot_asset_fingerprint.clj

```diff
@@ -27,8 +36,9 @@
                           (core/output-files)
                           (core/by-ext (or extensions [".html"])))
             file-hashes (into {}
-                          (map (juxt :path :hash))
-                          (core/output-files fileset))]
+                              (map (juxt :path :hash))
+                              (core/by-ext (or rename-extensions [".js" ".css"]) (core/output-files fileset)))
```

Comment:
It would add significant complexity, but it might be interesting to only rename files that actually had a fingerprint replacement done in some other file (i.e. that filename was found inside of a `${}`).

Then there would be no need for `rename-extensions` and fingerprinting any file would __Just Work__ ™️.